lib/internal/modules/esm/get_format.js

```diff
@@ -71,9 +69,19 @@ function defaultGetFormat(url, context, defaultGetFormatUnused) {
         throw new ERR_UNKNOWN_FILE_EXTENSION(ext, fileURLToPath(url));
       }
     }
-    return { format: format || null };
-  }
-  return { format: null };
+
+    return format || null;
+  },
+  'node:'() { return 'builtin'; },
+  noop() { return null; },
```

Comment:
```suggestion
```

Reply:
Did you have a reason behind this? I included it so get_format doesn't blow up trying to call a nonexistent function (which would otherwise happen with `protocolHandlers[parsed.protocol](parsed, url)`). `null` eventually gets detected in `ESMLoader.load()` and throws an error there (because the final value of `format` cannot be `null`). This null is a sort of flag that a value for format was never set nor was it determinable.

Reply:
I think you should just return `null` if the protocol is unknown instead of invoking a `noop` function (ref my suggestion with  `ObjectPrototypeHasOwnProperty`).

Reply:
```js
return protocolHandlers[parsed.protocol]?.(parsed, url)) ?? null;
```

Like this? I do like that it saves the function call but thought it looked more convoluted.

Reply:
Well, more like this 😅
```js
  return ObjectPrototypeHasOwnProperty(protocolHandlers, parsed.protocol) ?
    protocolHandlers[parsed.protocol](parsed, url) :
    null;
```